worknenjoy/gitpay
# Fix deadline update notification email

## Description

> This PR fixes the bug for email notification of deadline updates to assigned users

## Changes

- mail/deadline.js 

## Issue

> Closes #499

## Impacted Area

> DeadlineMail.update()  function

## Steps to test

Steps needed to reproduce the scenario from this change to validate if the pull request solve this issue

- Create an user
- Import an issue
- Create another user
- Apply to solve an issue in I'm interested
- Log with the first user
- Assign user
- Change deadline
- An email should be sent (locally will not receive an email, but you can debug on console)
